test/test_xnnpack_integration.py

```diff
@@ -934,6 +936,7 @@ def forward(self, x):
                      " XNNPACK must be enabled for these tests."
                      " Please build with USE_XNNPACK=1.")
 @unittest.skipIf(TEST_WITH_TSAN, "TSAN is not fork-safe since we're forking in a multi-threaded environment")
+@unittest.skipIf(TEST_WITH_ROCM, "HACK, not sure why these fail on ROCM.")
```

Comment:
This is ok for now. Do file a follow up task with details on which build failed and what you tried to resolve.

Reply:
Created T112777496.